src/sGLOH2.cpp

```diff
@@ -32,7 +32,7 @@
 constexpr int PATCH_SIZE = 41;
 constexpr int N = 2;
 constexpr int M = 4;
-constexpr int Q = 8;
+constexpr int Q = 4;
```

Comment:
I understand M, N, and PATCHSIZE - what is Q doing here though?

Reply:
I know we talked about it the other day, but I think I'm still not quite on the same page on that one

Reply:
I had q for the blocks and it has slowly turned into M going to remove this just wanted your testing to be on the current version. Should just be clean up and tweaking from now till turn in unless I find another part that really bothers me.

Reply:
Gotcha - that makes sense!